src/SFA.DAS.ApprenticeFeedback.Application.UnitTests/AutoFixtureExtensions.cs

```diff
@@ -60,23 +60,27 @@ public object Create(object request, ISpecimenContext context)
         {
             if (request is Type type && type == typeof(ApprenticeFeedbackDataContext))
             {
+                return GetApprenticeFeedbackDataContext();
+            }
 
-                var connection = new SqliteConnection("Filename=:memory:");
-                connection.Open();
+            return new NoSpecimen();
+        }
 
-                // These options will be used by the context instances in this test suite, including the connection opened above.
-                var contextOptions = new DbContextOptionsBuilder<ApprenticeFeedbackDataContext>()
-                    .UseSqlite(connection)
-                    .Options;
+        public static ApprenticeFeedbackDataContext GetApprenticeFeedbackDataContext()
+        {
+            var connection = new SqliteConnection("Filename=:memory:");
+            connection.Open();
 
-                // Create the schema and seed some data
-                var dbContext = new ApprenticeFeedbackDataContext(contextOptions);
-                dbContext.Database.EnsureCreated();
+            // These options will be used by the context instances in this test suite, including the connection opened above.
+            var contextOptions = new DbContextOptionsBuilder<ApprenticeFeedbackDataContext>()
+                .UseSqlite(connection)
+                .Options;
 
-                return dbContext;
-            }
+            // Create the schema and seed some data
+            var dbContext = new ApprenticeFeedbackDataContext(contextOptions);
+            dbContext.Database.EnsureCreated();
 
-            return new NoSpecimen();
+            return dbContext;
```

Comment:
This is simply putting some code into a static method, so that I can use it in a unit test that doesn't require all this automocking.